pkg/cmd/describe/origin/origin.go

```diff
@@ -60,7 +60,7 @@ func NewCmd(f *cmdutil.Factory) *cobra.Command {
 				applicationID = int64(appID)
 			}
 
-			if !cmd.Flags().Changed("origin-id") {
+			if !cmd.Flags().Changed("origin-key") {
 				answers, err := utils.AskInput("What is the ID of the Origin?")
```

Comment:
Put this message in a variable. Also, you have to ask about origin key and not origin ID.